src/gpu/collector.cpp

```diff
@@ -94,6 +100,7 @@ auto Collector<T>::serialize() const -> HostArray<char, 1> {
 
   for(std::size_t i = 0; i < wlData_.size(); ++i) {
     totalNumBytes += sizeof(SerialInfo<T>);
+    totalNumBytes += sizeof(std::size_t);
```

Comment:
Was this added because of `nVis`? If so, this should already be included in `SerialInfo`.

Reply:
Thanks for the catch. I fixed that.